src/Routes/Lists.js

```diff
@@ -278,6 +280,23 @@ class ListsRoute extends BaseRoute {
             }
         });
 
+        this.router.get('/best-practices', async (req, res) => {
+            try {
+                const bestPractices = await this.db.select().from('kv_cache').where({ key: 'list_best_practices' });
+                if (!bestPractices.length) return res.status(503).render('error', {
+                    title: 'Data not found',
+                    status: 503,
+                    message: 'Best practices has not yet been loaded.'
+                });
+                res.render('lists/best_practices', {
+                    title: 'Best Practices for Discord Bot Lists',
+                    data: this.renderer.render(bestPractices[0].value)
```

Comment:
Could we some logic to drop the `h1` title from the MD, as it's already rendered in the page itself?